src/core.js

```diff
@@ -1,3 +1,4 @@
+d3 = require('d3');
```

Comment:
What happens if we include this in the browser normally and 'require' isn't defined? Won't this error out?